index.html

```diff
@@ -13,35 +13,189 @@
+                </div>
+                
+            </li>
+            <li class="grid-item white"> 
+                <div class="content">
+                    <h2 class="title">Data Dashboard Healthcare</h2>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <div class="btn">
+                        <button type="button" class="grey-button">HTML</button>
+                        <button type="button" class="grey-button">Bootstrap</button>
+                        <button type="button" class="grey-button">Ruby</button>
+                    </div>
+                        <button type="button" class="orange-button">See project</button>
+                    </div>
+                    
+                </li>
+                <li class="grid-item white"> 
+                    <div class="content">
+                        <h2 class="title">Data Dashboard Healthcare</h2>
+                        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <div class="btn">
+                            <button type="button" class="grey-button">HTML</button>
+                            <button type="button" class="grey-button">Bootstrap</button>
+                            <button type="button" class="grey-button">Ruby</button>
+                        </div>
+                            <button type="button" class="orange-button">See project</button>
+                        </div>
+                        
+                    </li>
+                    <li class="grid-item white"> 
+                        <div class="content">
+                            <h2 class="title">Data Dashboard Healthcare</h2>
+                            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                            <div class="btn">
+                                <button type="button" class="grey-button">HTML</button>
+                                <button type="button" class="grey-button">Bootstrap</button>
+                                <button type="button" class="grey-button">Ruby</button>
+                            </div>
+                                <button type="button" class="orange-button">See project</button>
+                            </div>
+                            
+                        </li>
+
+    </ul>
+
+    </main>
+
+    <!-- About Section -->
+    <section id="about">
+        <div id="intro">
+        <h2 id="about-me" class="orange-color">About me</h2>
+        <p class="text">Hey there! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```

Comment:
- [ ] Here as well, please follow the figma design guidelines:

#### figma design:
![f4](https://user-images.githubusercontent.com/105105262/227730400-265667a5-eab8-41fb-9779-c6b0b38bc062.PNG)

#### your design:
![y4](https://user-images.githubusercontent.com/105105262/227730408-64d4bd21-2508-41d3-a728-6d6b05d4c445.PNG)

Reply:
- [ ] Great job, @sylvester-wamaya :wave: but you still have some adjustment to make in order to close this issue as accomplished. Kindly watch [this video recording](https://www.awesomescreenshot.com/video/15935431?key=0b74d405212a12bd15449a7b473f0525) I made specially for you. It will give you an overview of the corrections needed as regarding the missing demarcation line, `margin-top`, among other issues with your list of skills. :hammer_and_wrench: